sections/run-job.md

```diff
@@ -26,7 +26,7 @@ $ curl -X POST -u api_key: "https://api.xplenty.com/:account_id/api/jobs" \
     "dynamic_variables":{
       "current_time":"CurrentTime()",
       "MY_CURRENT_TIME":"$CURRENT_TIME_VAR",
-      "MY_STRING_VAR":"'some string'"
+      "MY_STRING_VAR":"some string"
```

Comment:
@motymichaely It generates wrong json. Quotation marks has to be represented as special character: `\u0027`.